doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,165 @@
-window*, where various messages resulting from commands are displayed.
-At the bottom is the status bar.
+The screen shot shows CoqIDE while the user is stepping through the file `Fermat.v`.
+A menu bar and tool bar appear at the top of the window. The left-hand panel shows
+the current *script buffer*.  Each script buffer corresponds to a separate Coq process.
+The upper right panel is the *goal panel*, which shows
+the goals to be proven.  The lower right panel has the *message
+panel*, which shows messages produced by commands and tactics.
+The contents of the right-hand panels are specific to the currently-displayed script.
+The status bar is at the bottom.
 
 Managing files and buffers, basic editing
-----------------------------------------------
-
-In the script window, you may open arbitrarily many buffers to edit.
-The *File* menu allows you to open files or create some, save them,
-print or export them into various formats. Among all these buffers,
-there is always one which is the current *running buffer*, whose name
-is displayed on a background in the *processed* color (green by default), which
-is the one where Coq commands are currently executed.
-
-Buffers may be edited as in any text editor, and classical basic
-editing commands (Copy/Paste, …) are available in the *Edit* menu.
-CoqIDE offers only basic editing commands, so if you need more complex
-editing commands, you may launch your favorite text editor on the
-current buffer, using the *Edit/External Editor* menu.
-
-Interactive navigation into Coq scripts
---------------------------------------------
-
-The running buffer is the one where navigation takes place. The toolbar offers
-five basic commands for this. The first one, represented by a down arrow icon,
-is for going forward executing one command. If that command is successful, the
-part of the script that has been executed is displayed on a background with the
-processed color. If that command fails, the error message is displayed in the
-message window, and the location of the error is emphasized by an underline in
-the error foreground color (red by default).
-
-In the figure :ref:`CoqIDE main screen <coqide_mainscreen>`,
+-----------------------------------------
+
+The *File* menu lets you open files, create new buffers, save buffers to a file,
+and print or export them into various formats.
+
+Text editing provides the basic operations such as copy/cut/paste and replace.
+Most editing operations are shown in the *Edit* menu, which shows the equivalent
+keystrokes for operations.  if you need more complex editor commands, you can
+launch your favorite text editor on the current buffer, using the
+*Edit/External Editor* menu. (When you're done editing you must currently must
+reopen the file to see your changes.)  Also note:
+
+- Undo is also available as Ctrl-Z.  Redo is Ctrl-Shift-Z.
+- Select all is Ctrl-A.
+- Commenting and uncommenting the current line or selected text is available in
+  the *Tools* menu.
```

Comment:
Ctrl (Linux and Windows) and Cmd for Mac I believe. Perhaps we should simply have a note somewhere about Mac users.

Reply:
Also commenting and commenting is mapped to Ctrl-D and depends on the line of the cursor and the text highlighted.

Reply:
We should find a Mac user to verify this.  I didn't mention Ctrl-D because the mapping is discoverable from the menu (while Ctrl-Z and Ctrl-A don't appear on any menu).

Reply:
I think Ctrl-A and Ctrl-Z may be hard coded into the script panel widget.

Per https://support.apple.com/en-us/HT201236, these should be equivalent to Command-A and Command-Z on macos.

FWIW, we need a documented user-friendly way to configure key mappings.  I couldn't get https://github.com/coq/coq/wiki/Configuration-of-CoqIDE to work so far.  You've seen the Edit/Preferences/Shortcuts panel (which works)?  But not in this PR.